### PR TITLE
Planting quotes & theme

### PR DESCRIPTION
**Pull request by mecathcart**
updated theme

Screenshot 2024-03-27 at 11 05 51 PM
Screenshot 2024-03-27 at 11 05 40 PM

Screenshot 2024-03-27 at 11 07 13 PM
Screenshot 2024-03-27 at 11 06 00 PM

Screenshot 2024-03-27 at 11 12 29 PM

Screenshot 2024-03-27 at 11 10 51 PM

### CODE REVIEW

File: src/components/modals/InfoModal.tsx [hunk 1]
```diff
@@ -68,15 +68,18 @@ export const InfoModal = ({ isOpen, handleClose }: Props) => {
...
       <p className="text-sm text-gray-500 dark:text-gray-300">
-        The letter P is in correct spot.
+        The letter 'P' is indicated by a green tile when it's in the correct
+        spot.
       </p>
 
       <p className="text-sm text-gray-500 dark:text-gray-300">
-        The letter L is in the message but in the wrong spot.
+        The letter 'L' is represented by a yellow tile to indicate it's in the
+        message but placed incorrectly.{' '}
```
Comment (cesine) [inline src/components/modals/InfoModal.tsx:77]: what is the `{' '}` for?

```suggestion
        message but placed incorrectly.
```

File: src/index.css [hunk 2]
```diff
@@ -9,7 +9,7 @@
   --default-cell-border-color: theme('colors.black');
   --default-cell-text-color: theme('colors.black');
   --absent-cell-bg-color: theme('colors.slate.400');
-  --correct-cell-bg-color: theme('colors.green.400');
+  --correct-cell-bg-color: theme('colors.Darkgreen');
```
Comment (cesine) [inline src/index.css:12]:
```suggestion
  --correct-cell-bg-color: theme('colors.green.800');```

File: tailwind.config.js [hunk 3]
```diff
@@ -2,7 +2,18 @@ module.exports = {
   content: ['./src/**/*.{ts,tsx}'],
   darkMode: 'class',
   theme: {
+    
+
     extend: {
+      
+      colors: {
+        'Darkgreen': '#1E6E26',
```
Comment (cesine) [inline tailwind.config.js:10]:
```suggestion
```

File: src/components/navbar/Navbar.tsx [hunk 4]
```diff
@@ -120,42 +121,56 @@ export const Navbar = ({
               }}
               title="Click to send me a bug report"
             />
+            <span className="ml-3 mt-1 text-xs">Report</span>
           </a>
-          <ChatBubbleOvalLeftIcon
+          <a
```
Comment (cesine) [inline src/components/navbar/Navbar.tsx:126]: This isnt a link, if you click on this, it opens anothe tab, which is confusing

File: src/components/modals/InfoModal.tsx [hunk 5]
```diff
@@ -68,15 +68,18 @@ export const InfoModal = ({ isOpen, handleClose }: Props) => {
       </div>
 
       <p className="text-sm text-gray-500 dark:text-gray-300">
-        The letter P is in correct spot.
+        The letter 'P' is indicated by a green tile when it's in the correct
```
Comment (cesine) [inline src/components/modals/InfoModal.tsx:71]: too many words

Screenshot 2024-03-27 at 9 25 45 PM

Comment (mecathcart) [reply src/components/modals/InfoModal.tsx:71]: agreed, I will reword

File: src/components/navbar/Navbar.tsx (hunk 4 above)
Comment (mecathcart) [reply src/components/navbar/Navbar.tsx:126]: can you disreagard those commits? I was just trying stuff to see what it'd look like.

Comment (cesine) [reply src/components/navbar/Navbar.tsx]: its okay i figured out what you were doing and updated it so that it matches the goal

**Pull request merged**